### ISSUE TITLE
Problem Creating Adonis Project

### ISSUE THREAD

**Issue by darmie** [adonisjs/adonis-cli#2]
I am having this error when I create Adonis project. Please fix.

```
npm ERR! Windows_NT 10.0.10586
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "--production"
npm ERR! node v4.2.1
npm ERR! npm  v2.14.7
npm ERR! code EBADPLATFORM

npm ERR! notsup Unsupported
npm ERR! notsup Not compatible with your operating system or architecture: fsevents@1.0.6
npm ERR! notsup Valid OS:    darwin
npm ERR! notsup Valid Arch:  any
npm ERR! notsup Actual OS:   win32
npm ERR! notsup Actual Arch: x64

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\Damilare\freelance\Kangpe\npm-debug.log
```

**Comment by thetutlage** [comment]
Share following your windows version

**Comment by darmie** [comment]
Windows 10, x64

**Comment by darmie** [comment]
@thetutlage  Please I am still having this problem. Can you suggest how to fix it?

**Comment by thetutlage** [comment]
the problem is with fsevents, let me check it out by installing windows.

**Comment by darmie** [comment]
I just found out that it's an optional dependency. I guess it won't affect
anything.

**Comment by RomainLanz** [comment]
@darmie Can you try to run `adonis fix` into your project?

**Comment by RomainLanz** [comment]
@thetutlage This can be closed.

**Comment by Ifnot** [comment]
I still get this problem in a Windows 10 64bits (with last update).

I just install the `adonis-cli` with `npm install -g anodis-cli`, and create a new project with `adonis new my_project`. The git clone is good, the npm install fails at the end with the exactly same error above.

Running `npm start` after the installation run the project well but running `npm install` manually fails.

I think it is not a `adonis-cli` related bug but a included package in `adonis-framework` related bug but i reopened here since it is the first result i found on google.

`node -v v4.4.5`
`npm -v 2.15.5`

**Comment by RomainLanz** [comment]
Please, upgrade `npm`

**Comment by Ifnot** [comment]
Wow did not noticed that it was really outdated on my windows. I am up to date now.

But the problem is still here :

```
npm ERR! Windows_NT 10.0.10586
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install"
npm ERR! node v4.4.5
npm ERR! npm  v3.9.6
npm ERR! code ELIFECYCLE

npm ERR! fsevents@1.0.6 install: `node-pre-gyp install --fallback-to-build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the fsevents@1.0.6 install script 'node-pre-gyp install --fallback-to-build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the fsevents package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-pre-gyp install --fallback-to-build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs fsevents
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls fsevents
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\anael\CloudStation\LianLi\Bureau\Project_Cars\npm-debug.log
```

Thanks for your fast response btw :)

**Comment by GTCrais** [comment]
I have the same problem:
```
54673 verbose stack Error: fsevents@1.0.14 install: `node-pre-gyp install --fallback-to-build`
54673 verbose stack Exit status 1
54673 verbose stack     at EventEmitter.<anonymous> (C:\Users\Crais\AppData\Roaming\npm\node_modules\npm\lib\utils\lifecycle.js:242:16)
54673 verbose stack     at emitTwo (events.js:87:13)
54673 verbose stack     at EventEmitter.emit (events.js:172:7)
54673 verbose stack     at ChildProcess.<anonymous> (C:\Users\Crais\AppData\Roaming\npm\node_modules\npm\lib\utils\spawn.js:40:14)
54673 verbose stack     at emitTwo (events.js:87:13)
54673 verbose stack     at ChildProcess.emit (events.js:172:7)
54673 verbose stack     at maybeClose (internal/child_process.js:827:16)
54673 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
54674 verbose pkgid fsevents@1.0.14
54675 verbose cwd C:\Users\Crais\testproject
54676 error Windows_NT 10.0.10586
54677 error argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Crais\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "install"
54678 error node v4.4.7
54679 error npm  v3.10.5
54680 error code ELIFECYCLE
54681 error fsevents@1.0.14 install: `node-pre-gyp install --fallback-to-build`
54681 error Exit status 1
54682 error Failed at the fsevents@1.0.14 install script 'node-pre-gyp install --fallback-to-build'.
54682 error Make sure you have the latest version of node.js and npm installed.
54682 error If you do, this is most likely a problem with the fsevents package,
54682 error not with npm itself.
54682 error Tell the author that this fails on your system:
54682 error     node-pre-gyp install --fallback-to-build
54682 error You can get information on how to open an issue for this project with:
54682 error     npm bugs fsevents
54682 error Or if that isn't available, you can get their info via:
54682 error     npm owner ls fsevents
54682 error There is likely additional logging output above.
54683 verbose exit [ 1, true ]
```

**Comment by GTCrais** [comment]
Solved it - downgrading to npm version 3.8.* fixed this issue. Found the answer here:
https://github.com/adonisjs/adonis-framework/issues/161
